src/common/utils/colors.ts

```diff
@@ -0,0 +1,19 @@
+const colors = {
+  lifeBlue: '#02BFF1',
+  backgroundLightGray: '#EFEFF4',
+  backgroundSeenNotification: '#F1F1F1',
+  disabledText: '#959AB4',
+  destructiveRed: '#FD2727',
+  warningYellow: '#FECA2F',
+  textLightBlue: '#66DFFF',
+  facebookBlue: '#4267b2',
+  highlight: '#F2F2F6',
+  white: '#FFFFFF',
+
+  announcement: {
+    gradientOrangeMedium: '#F2994A',
+    gradientOrangeLight: '#F2C94C',
+  },
+} as const;
+
+export default colors;
```

Comment:
I've been trying to avoid this file in hopes that we can introduce it as part of the eventual `useBranding` PR.  Is it possible to remove this, or does the component look ridiculous without it?

Reply:
I will give it a shot and see what it looks like.